code/RedirectedURLCsvBulkLoader.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+class RedirectedURLCsvBulkLoader extends CsvBulkLoader
+{
+    private $exception_thrown;
+
+    /**
+     * @todo Better messages for relation checks and duplicate detection
```

Comment:
Is this `@todo` actually relevant? What about the rest of this docblock?

Reply:
I just copied that doc from parent class's func. Not 100% sure what I need to do with that. Could you please suggest?

Reply:
I'd remove anything that you don't think is relevant :)

Reply:
Removed that unwanted `@todo`